src/LogMonitorEntry.js

```diff
@@ -42,7 +43,7 @@ export default class LogMonitorEntry extends Component {
             theme={this.props.theme}
             keyName={'state'}
             data={this.props.select(state)}
-            previousData={this.props.select(this.props.previousState)}
+            previousData={this.props.select(this.previousState)}
```

Comment:
Why not `previousData={typeof this.props.state !== 'undefined' ? this.props.select(this.props.state) : undefined}`?